storybook/stories/Button/Button.test.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React from 'react';
+import { Text } from 'react-native';
+import { mount } from 'enzyme';
+import Button from './index';
+
+describe('Button', () => {
+  it('renders witout crashing', () => {
+    const component = (
+      <Button />
+    );
+    mount(component);
+  });
+  it('clicks', () => {
+    const onPressCallback = jest.fn();
+    const component = (
+      <Button onPress={onPressCallback} />
+    );
+    const button = mount(component);
+    // yeah, I know it doesn't make any sense but enzyme for RN doesn't support .simulate('press')
```

Comment:
Not sure about this, but if we can't simulate the click, this is not really testing anything, since we're just calling the function we just passed to it via props :thinking:

Reply:
Yeah it's not really testing, that's why I added this comment. Those tests are just to check if testing is working, these components will be removed anyway after I add real components.